www-servers/nginx-unit/nginx-unit-1.3.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not 7?

Reply:
I used `/usr/portage/skel.ebuild` as a template, and it comes with EAPI=6. I can of course change this.